### DISCUSSION TITLE
Is there a way to determine if a value is included in an array?

### DISCUSSION THREAD

**Discussion by ElixirMike** [handsontable/hyperformula#1148] [API Questions]
I'm just learning the basics of hyperformula and trying to see if it will work with some of our requirements.
In our case, we will need the ability to work with arrays of data . We are thinking about using just passing in what data we need evaluated and run a one-off calculation.

Consider that we have an array of data with  {"one","two"}.
I now want to run a condition to check if "one" exists in the array.
I 'm trying to use the MATCH function as I can't find anothe r function that would be like an INCLUDES or CONTAINS.

const result=hfInstance.calculateFormula(`=MATCH("one",{"one","two"},"Text")`,0)
This throws error

DetailedCellError {
  value: '#VALUE!',
  address: undefined,
  type: 'VALUE',
  message: 'Value cannot be coerced to number.'
}

Is there a way to check if a value is in an array? Are there functions that would let me retrieve a specific value in the array?

Thanks for your help!

**Comment by sequba** [comment]
Hi @ElixirMike, thanks for reaching out to us.

The third parameter of the `MATCH()` function call must be a number `-1`, `0` or `1` as described [by the Excel documentation](https://support.microsoft.com/en-us/office/match-function-e8dffd45-c762-47d6-bf89-533f4a37673a). Most of functions in HyperFormula works the same as in other popular spreadsheet software.

```js
const result = hf.calculateFormula('=MATCH("one", {"one","two"}, 0)', 0);
console.log(result);
```

**Comment by ElixirMike** [comment]
ah, on your documentation, it just listed "Type", so I thought that meant data type. 
Thx for response.
